recipe/meta.yaml

```
@@ -11,23 +11,23 @@ source:
   sha256: e9cd98c2bb35bf414fe721469a043cdcc10f0808d1ffcf606acb4a6282a6f288
 
 build:
+  skip: true  # [py<37]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - flit-core 3.3.0
```

```suggestion
    - flit-core >=3.2,<4
```
To ease building for other python versions.